### PR TITLE
Modified the App with the Proper Design Patterns.

### PR DESCRIPTION
**Pull request by mhamzajutt96**

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -1,55 +1,46 @@
...
-    def index
-        @orders = current_user.orders
+	def index
+		@orders = current_user.orders
+		if params[:checkout] == 'success'
+			flash[:notice] = 'Payment is Successful'
+		end
+	end
+	
+	def create_session
```
Comment (17100262) [inline app/controllers/orders_controller.rb:14]: The purpose to implement the service object was to encapsulate the business logic into a single class. That purpose is still haven't achieved. This is part of business logic

```
if @order.save
	@stripe_adapter = StripeAdapter.new(@product, @order)
	@session = @stripe_adapter.make_session
	f.js {}
else
	f.js {}
end
```

File: app/services/order_generator.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class OrderGenerator
+  
+  def self.generate_order(user, product)
+    user_id = user.id
+    @product = product
```
Comment (17100262) [inline app/services/order_generator.rb:5]: Why haven't you used the constructor for such initialization?

File: app/services/order_generator.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class OrderGenerator
+  
+  def self.generate_order(user, product)
```
Comment (17100262) [inline app/services/order_generator.rb:3]: Why is this a class function?

File: app/services/order_generator.rb (hunk 2 above)
Comment (17100262) [inline app/services/order_generator.rb:5]: You have implemented class method but used instance variable which is very odd.

Comment (17100262) [reply app/services/order_generator.rb:5]: Constructors are usually used to set such values

Comment (17100262) [reply app/services/order_generator.rb:5]: No proper class implementation as well as no result object implemented

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (mhamzajutt96) [reply app/controllers/orders_controller.rb:14]: Now the evaluation is over, Kindly provide the solution for every of the mistake you mentioned.

**Pull request closed without merging**